include/swift/AST/IRGenOptions.h

```diff
@@ -168,18 +168,20 @@ class IRGenOptions {
   /// Emit mangled names of anonymous context descriptors.
   unsigned EnableAnonymousContextMangledNames : 1;
 
-  /// Enables resilient class layout.
-  unsigned EnableClassResilience : 1;
-
   /// Bypass resilience when accessing resilient frameworks.
   unsigned EnableResilienceBypass : 1;
 
   /// Force lazy initialization of class metadata
   /// Used on Windows to avoid cross-module references.
   unsigned LazyInitializeClassMetadata : 1;
 
+  /// Normally if the -read-legacy-type-info flag is not specified, we look for
+  /// a file named "legacy.yaml" in the same directory as Swift.swiftmodule.
+  /// Passing this flag completely disables this behavior.
+  unsigned DisableLegacyTypeInfo : 1;
+
   /// The path to load legacy type layouts from.
-  StringRef ReadTypeInfoPath;
+  StringRef ReadLegacyTypeInfo;
```

Comment:
This is still a path, no? Right now this doesn't sound like the name of a string. (Neither does the new command-line option name.)

Reply:
I'm going to rename it to `-read-legacy-type-info-path`, but I'm keeping `-disable-legacy-type-info`.